### ISSUE TITLE
Speed/accuracy trade-offs for modern convolutional object detectors

### ISSUE THREAD

**Issue by leo-p** [leo-p/papers#37]
https://arxiv.org/pdf/1611.10012.pdf

In this paper, we study the trade-off between accuracy and speed when building an object detection system based on convolutional neural networks. We consider three main families of detectors — Faster R-CNN, R-FCN and SSD — which we view as “meta-architectures”. Each of these can
be combined with different kinds of feature extractors, such as VGG, Inception or ResNet. In addition, we can vary other parameters, such as the image resolution, and the number of box proposals. We develop a unified framework (in Tensorflow) that enables us to perform a fair comparison between all of these variants. We analyze the performance of many different previously published model combinations, as well as some novel ones, and thus identify a set of models which achieve different points on the speed-accuracy tradeoff curve, ranging from fast models, suitable for use on a mobile phone, to a much slower model that achieves a new state of the art on the COCO detection challenge.

**Comment by leo-p** [comment]
## Summary:
- *Objective:* Compare several meta-architectures and hyper-parameters in the same framework for easy comparison.

## Architectures:
Four meta architectures:
1. R-CNN
2. Faster R-CNN
3. SSD
4. YOLO Architecture (not evaluated in the paper)

<img width="935" alt="screen shot 2017-05-05 at 3 12 57 pm" src="https://cloud.githubusercontent.com/assets/17261080/25746807/5a294360-31a5-11e7-808e-d48497a16cd5.png">

## Results:
Very interesting to know which framework to implement or not at first glance.
